topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,441 @@
+---
+layout: tutorial_hands_on
+
+title: Trio Analysis using Synthetic Datasets from RD-Connect GPAP
+zenodo_link: 'https://sandbox.zenodo.org/record/1055250'
+questions:
+- How do you import data from the EGA?
+- How to download files with HTSGET in Galaxy?
+- How do you pre-process VCFs?
+- How do you identify causative variants?
+objectives:
+- Requesting DAC access and importing data from the EGA.
+- Pre-process VCFs using regular expressions and AWK.
+- Use annotations and phenotype information to find the causative variant(s).
+time_estimation: 2H
+key_points:
+- Downloading whole datasets with HTSGET is safe and easy with galaxy.
+- AWK and regex are usefull tools for pre-processing VCF files.
+- Variant annotations allows us to strictly filter VCFs to find the causative variant.
+contributors:
+- Jasper Ouwerkerk
+
+---
+
+# Introduction
+{:.no_toc}
+To discover causal mutations of inherited diseases it’s common practice to do a trio analysis. In a trio analysis DNA is sequenced of both the patient and parents. Using this method, it’s possible to identify multiple inheritance patterns. Some examples of these patterns are autosomal recessive, autosomal dominant, and  de-novo variants, which are represented in the figure below. To elaborate, the most left tree shows an autosomal dominant inhertitance pattern where the offspring inherits a faulty copy of the gene from one of the parents. The center subfigure represents an autosomal recessive disease, here the offspring inherited a faulty copy of the same gene from both parents. In the right subfigure a de-novo mutation is shown, which is caused by a mutation during the offspring’s lifetime.
+
+| ![plot](../../images/trio-analysis/pedigree.svg) |
+|:--:|
+| **Figure 1:** *3 family trees representing autosomal dominant, autosomal recessive, and a de-novo inheritence patter from parents to son.* |
+
+To discover these mutations either whole exome sequencing (WES) or whole genome sequencing (WGS) can be used. With these technologies it is possible to uncover the DNA of the parents and offspring to find (shared) mutations in the DNA. These mutations can include insertions/deletions (indels), loss of heterozygosity (LOH), single nucleotide variants (SNVs), copy number variations (CNVs), and fusion genes. 
+
+In this tutorial we will also make use of the HTSGET protocol, which is a program to download our data securely and savely. This protocol has been implemented in the {% tool [EGA Download Client](toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.0+galaxy0) %} tool, so we don't have to leave galaxy to retrieve our data.

```

Comment:
Make it clear in these paragraphs that this tutorial demonstrates how to obtain **variant information** from an online archive, not the original sequencing data (because that is the real difference compared to our existing tutorial).

Reply:
I've made it more clear in 2ca1e55afaade48fa3eb694b32f59b93f8d468f2